tests/bluetooth/bsim_bt/bsim_test_mesh/prj.conf

```diff
@@ -40,5 +40,6 @@ CONFIG_BT_MESH_PROVISIONER=y
 CONFIG_BT_MESH_PROV_DEVICE=y
 CONFIG_BT_MESH_CDB=y
 CONFIG_BT_MESH_CDB_NODE_COUNT=4
+CONFIG_BT_MESH_PROV_OOB_PUBLIC_KEY=y
```

Comment:
Will this affect any other test suites?

Reply:
No. If public/private keys pair is not initialized then provisioning goes in normal way.